build-support/bin/native/bootstrap.sh

```diff
@@ -15,11 +15,12 @@ source ${REPO_ROOT}/build-support/common.sh
 source ${REPO_ROOT}/build-support/bin/native/detect_os.sh
 
 readonly NATIVE_ROOT="${REPO_ROOT}/src/rust/engine"
-readonly MODE=debug
-readonly MODE_FLAG=
-
 readonly NATIVE_ENGINE_VERSION_RESOURCE="${REPO_ROOT}/src/python/pants/engine/subsystem/native_engine_version"
 
+# N.B. Set $MODE to "debug" to generate a binary with debugging symbols.
```

Comment:
ah, poor assumption on my part due to not having first class access to `cargo` since internalization - fixed!